src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudConfig.scala

```diff
@@ -1,31 +1,29 @@
 package org.broadinstitute.dsde.firecloud
 
-import scala.jdk.CollectionConverters._
-import com.typesafe.config.{ConfigFactory, ConfigObject}
-import org.broadinstitute.dsde.firecloud.service.{FireCloudDirectiveUtils, NihWhitelist}
-import org.broadinstitute.dsde.rawls.model.{EntityQuery, SortDirections}
-import org.broadinstitute.dsde.workbench.model.{WorkbenchEmail, WorkbenchGroupName}
-
 import akka.http.scaladsl.model.Uri
 import akka.http.scaladsl.model.Uri.{Authority, Host, Query}
+import com.typesafe.config.{Config, ConfigFactory, ConfigObject}
+import org.broadinstitute.dsde.firecloud.service.{FireCloudDirectiveUtils, NihWhitelist}
+import org.broadinstitute.dsde.rawls.model.{EntityQuery, SortDirections}
+import org.broadinstitute.dsde.workbench.model.WorkbenchGroupName
 
+import scala.jdk.CollectionConverters._
 import scala.util.Try
 
 object FireCloudConfig {
   private val config = ConfigFactory.load()
 
   object Auth {
     private val auth = config.getConfig("auth")
-    val googleClientId = auth.getString("googleClientId")
-    val googleSecretJson = auth.getString("googleSecretsJson")
-
+    // OIDC configuration
+    val authorityEndpoint = auth.getString("authorityEndpoint")
+    val oidcClientId = auth.getString("oidcClientId")
+    val oidcClientSecret = auth.optionalString("oidcClientSecret")
+    val extraGoogleClientId = auth.optionalString("extraGoogleClientId")
```

Comment:
Do we want some comment here around what extraGoogleClientId means?  (I'm guessing this is the one use when logging in with regular oauth?)

Reply:
Yeah I can add comments. This is the google clientId in Swagger for regular `implicit` flow. I wanted to make sure we can still use `googleoauth` in Swagger while we roll out B2C. I think this is temporary because eventually everything will use the same B2C client id (and we can remove the Google option).

Reply:
so basically `oidcClientId` is for B2C and `extraGoogleClientId` for the old Google OAuth flow right?

Reply:
Correct. I like `legacyGoogleClientId`, that name is probably more descriptive. Will update